pyne/srim_read.py

```diff
@@ -0,0 +1,165 @@
+"""This module is for SRIM files. Currently defines a
+SRIM class, reads a srim file.
+"""
+
+from warnings import warn
+from pyne.utils import QAWarning
+import numpy as np
+
+warn(__name__ + " is not yet QA compliant.", QAWarning)
+
+
+class srim(object):
+    """srim class includes srim specific variables"""
```

Comment:
This docstring doesn't help users who don't know what srim is. Could you expand please? Thanks!